cloud/scope/machine_pool.go

```diff
@@ -182,7 +182,9 @@ func (m *MachinePoolScope) SetListandSetMachinePoolInstances(ctx context.Context
 	providerIDList := make([]string, len(poolInstanceSummaries))
 
 	for i, instance := range poolInstanceSummaries {
-		providerIDList[i] = fmt.Sprintf("oci://%s", *instance.Id)
+		if *instance.State == "Running" {
```

Comment:
Great catch! should we use this instead `core.InstanceLifecycleStateRunning`? https://github.com/oracle/oci-go-sdk/blob/17bbd776e290d540c7a2ee08b66d27192e2f5ddb/core/instance.go#L357